iceoryx_hoofs/README.md

```diff
@@ -39,112 +22,136 @@ and we do not use dynamic memory. In these cases we adjusted the API to our use
-|`TACO`                    | i        | Thread Aware exChange Ownership (TACO). Solution if you would like to use `std::atomic` with data types larger than 64 bit. Wait free data synchronization mechanism between threads.       |
+| class                              | internal | description                                                                                                                                                                                                                        |
+|:----------------------------------:|:--------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|`FiFo`                              | i        | Single producer, single consumer lock-free FiFo                                                                                                                                                                                    |
+|`LockfreeQueue`                     |          | Multi producer, multi consumer lock-free FiFo with ringbuffer like overflow handling                                                                                                                                               |
+|`LoFFLi`                            | i        | Lock-free LIFO based index manager (lock-free free list). One building block of our memory manager. After construction it contains the indices {0 ... n} which you can acquire and release.                                        |
+|`SoFi`                              | i        | Single producer, single consumer lock-free safely overflowing FiFo (SoFi).                                                                                                                                                         |
+|`ResizeableLockFreeQueue`           |          | Resizeable variant of the `LockfreeQueue`                                                                                                                                                                                          |
+|`stack`                             |          | Stack implementation with simple push/pop interface.                                                                                                                                                                               |
+|`VariantQueue`                      |          | A queue which wraps multiple variants of Queues (FiFo, SoFi, ResizeableLockFreeQueue)                                                                                                                                              |
 
-attribute overview of the available Queues:
+#### Attribute overview of the available buffers
```

Comment:
The stack is missing here as well as the VariantQueue.

Reply:
@elfenpiff We can add the `stack` in a follow-up PR. The `VariantQueue` will be moved to `iceoryx_posh`.

Reply:
@mossmaurice in my opinion follow-up PRs do not work. It is very hard to track this discussion here in a completely different PR. And even when you link this with the issue and the next PR and is very cumbersome and confusing and even more time consuming that you adding the two lines with stack and variant queue.

In my opinion all the documentation on master should reflect the complete current state and not some possible future state.